### DISCUSSION TITLE
[Feature] Umbrel support

### DISCUSSION THREAD

**Discussion by rpjcbs** [immich-app/immich#2485] [Feature Request]
### The feature

Just installed Immich on my home Umbrel, set up fine but noticed it's running v1.40.0.

Downloaded the app on my phone and tried to connect it but getting no luck using http and https.

Is there anyway to manually update the app on my Umbrel?

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by samip5** [comment]
I think it should be directed towards the person that added it there. https://github.com/getumbrel/umbrel-apps/pull/239 as it's using quite the out-of-date docker compose file.

**Reply by rpjcbs** [reply]
Thanks, I've just commented on there as well. Apologies if this was the wrong place to post this.

**Reply by samip5** [reply]
It seems it shouldn't be too hard to update there though with mostly just copying the upstream docker compose file.

**Reply by rpjcbs** [reply]
It's a bit beyond my remit tbh, and I've just had a look around Github and can see some activity on [here](https://github.com/getumbrel/umbrel-apps/pull/526) which is great news!
